<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cercaPerIDDialog</class>
 <widget class="QDialog" name="cercaPerIDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreriaDigitale</string>
  </property>
  <widget class="QLabel" name="cercaIDlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>260</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Inserire ID dell'articolo che si vuole noleggiare:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cercaIDTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cercaIDButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius: 8px;
background-color:#333333; 
border: none;
color: white;
padding: 2px;
text-align: center;
font-size: 15px;
font-family: &quot;Garamond&quot;;
}

:hover {
background-color: #676767;
}

:pressed{
border: 1px solid #E0E2E2;
}</string>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cercaIDEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>220</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 10pt;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cercaIDErrorLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>260</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
